network_wrangler/params.py

```diff
@@ -165,19 +165,25 @@ def unique_ids(self) -> list[str]:
     "applied_projects",
     "conflicts",
 ]
-TRANSIT_CARD_TYPES = ["transit_property_change"]
+TRANSIT_CARD_TYPES = [
+    "transit_property_change",
+    "transit_routing_change",
+    "transit_route_addition",
+    "transit_service_deletion"
+]
 ROADWAY_CARD_TYPES = [
     "roadway_deletion",
     "roadway_addition",
     "roadway_property_change",
+    "pycode"
 ]
 SECONDARY_TRANSIT_CARD_TYPES = ["roadway_deletion"]
 
 # ---------------------------------------------------------------------
 # ------ SEARCH PARAMS ------
 # ---------------------------------------------------------------------
 
-DEFAULT_SEARCH_MODES = ["drive"]
+DEFAULT_SEARCH_MODES = ["drive","walk","bike"]
```

Comment:
is it okay to add walk, and bike modes to DEFAULT_SEARCH_MODES, otherwise some bike and walk links can't be selected and deleted

Reply:
The default is only used when you don't select something yourself. I think we want to limit default to be single mode.

Reply:
The issue is we don't have mode coded in the project cards, e.g. a roadway_deletion project card deletes hundreds of links, some of them are only bike or walk accessible. When it gets applied, those bike and walk links can't be selected and deleted.
Is there a good way to solve this issue? or should we manually add mode: ["walk", "bike", "drive"] to each roadway_deletion project cards. Cube/Metcouncil wrangler don't write out mode variable as well.

Reply:
we will add modes to project cards

Reply:
Hmmm - thinking this thru a bit more, wondering if for all deletion projects we should just default to "all" links?  I can't think of a time when we would only want to delete a link if it was walk_only or drive_access, etc.